Deploy/Build.ps1

```diff
@@ -7,7 +7,7 @@ param (
 
 $Output = $RootPath + '\Deploy\ADMU.ps1'
 $FormPath = $RootPath + '\jumpcloud-ADMU\Powershell\Form.ps1'
-$VersionRegex = [regex]'(?<=Title="JumpCloud ADMU )(.*?)(?=" )'
+$VersionRegex = [regex]'(?<=Title="JumpCloud ADMU )([0-9]+)\.([0-9]+)\.([0-9]+)'
```

Comment:
Just an update to conform to the other regex patterns in other tests/ build files. Semantic versioning regex pattern